user-management/src/main/java/de/tum/in/www1/artemis/usermanagement/config/ApplicationProperties.java

```diff
@@ -0,0 +1,13 @@
+package de.tum.in.www1.artemis.usermanagement.config;
```

Comment:
What does this class provide since it's empty?

Reply:
Nothing really, removed it